tendrl_node_agent/etcd_store.py

```diff
@@ -0,0 +1,23 @@
+import tendrl_node_agent.utils as utils
+import etcd
+from tendrl_node_agent.utils import Logger
+
+LOG = Logger().getLogger()
+
+
+@utils.singleton
+class EtcdStore():
+    def __init__(self):
+        configs = utils.TendrlConfig()
+        host_name = configs.get("general", "etcd_server")
```

Comment:
Locally I have the field names as used here :) Probably we need to finalize on its name due to conf file being shared by everything under tendrl umbrella. Probably, I'll use the same names as in already merged pieces in other repos.